app/build.gradle

```diff
@@ -53,4 +53,5 @@ dependencies {
     implementation 'de.hdodenhof:circleimageview:3.0.0'
     implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0'
     implementation 'com.github.felHR85:UsbSerial:6.0.6'
+    implementation 'org.jfree:jfreechart:1.0.14'
```

Comment:
We'll have to change these with "Android" compatible versions when we progress with real data.